src/prime_filed.cpp

```diff
@@ -57,13 +57,9 @@ int main(int argc, char** argv) {
   std::tie(drain_seconds, shutdown_seconds) = parse_quiesce_config(argc > 3 ? argv[3] : "");
   quiesce(drain_seconds, shutdown_seconds);
 
-  // default to no health check, if one is provided its just the path and the canned response is OK
-  http_server_t::health_check_matcher_t health_check_matcher{};
-  std::string health_check_response;
+  shortcircuiter_t<http_request_t> shortcircuiter;
   if (argc > 4) {
-    health_check_matcher = [&argv](const http_request_t& r) -> bool { return r.path == argv[4]; };
-    // TODO: make this configurable
-    health_check_response = http_response_t{200, "OK"}.to_string();
+    shortcircuiter = make_shortcircuiter(255, argv[4]);
```

Comment:
since we supported the healthcheck previously, it should be the first argument and yeah we should parse another arg to get the mask to use for the verbs. i would suggest some simple string thing `VERB,VERB,VERB` then you can split the string by `,` and h ave a simple map to look up which verb is which part of the mask. i would add this as a funciton to the http protocol, probably in http_util.h

Reply:
I think i could solve it!